conf/supervisord.conf.jnj

```diff
@@ -257,10 +257,8 @@ environment=
   PYTHONPATH=%(ENV_QUAYDIR)s,
   CONFIG_APP_PASSWORD=%(ENV_CONFIG_APP_PASSWORD)s,
   ENV_OPERATOR_ENDPOINT=%(ENV_OPERATOR_ENDPOINT)s,
-  CONFIG_EDITOR_STATIC_CONTENT_PATH=%(ENV_QUAYDIR)s/config_app/static,
-  MY_POD_NAMESPACE=must_fix,
```

Comment:
Will these be passed to the config editor backend?

Reply:
MY_POD_NAME and MY_POD_NAMESPACE can be removed since they are set by the operator and are not required by regular Quay instances. CONFIG_EDITOR_STATIC_CONTENT_PATH is set below. That line shows as removed since the comma was removed.

Reply:
these changes are tested and working